src/app/containers/Video/index.jsx

```diff
@@ -43,16 +47,29 @@ const VideoContainer = ({ blocks }) => {
     ['blocks', 1, 'model', 'blocks', 0, 'model', 'locator'],
     aresMediaBlock.model,
   );
-  const items = [
-    {
-      versionID,
-      kind,
-      duration,
+  const env = process.env.APP_ENV;
+  const mediaPlayerSettings = {
+    product: 'news',
```

Comment:
There's also an `insideIframe` option to indicate to the player whether it's inside an iframe, in the event it needs to navigate to another page. Do we think we need to set it? @sareh

Reply:
That prop `insideIframe` has its default is set to `false` already, according to the SMP cookbook documentation `http://cookbook.tools.bbc.co.uk/news/customise`. 
Yes, the issue should be updated to include this prop - we'll need to set the value of `insideIframe` to `true` when using it for AMP, within an iframe.